doc/usecases/done/gridlayer.md

```diff
@@ -0,0 +1,21 @@
+# Use Case: Grid
+
+## Summary
+
+- **Scope:** Grid Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The app will display a fixed scale coordinate grid.
+- **Status:** Done
+- **Assignee:** Moritz
+
+## Scenarios
+
+- **Precondition:**
+    - User has opened the map.
+- **Main success scenario:**
+    - The user sees a fixed scale coordinate grid with 1 meter spacing.
+    - Dots that indicate 10 m are bigger.
+- **Non-functional Constraints:**
+    - Support for changing viewport (zoom, position, etc.)
+    - The functionality is only available in the frontend.
```

Comment:
Links to implementation, protocol, ... missing

Reply:
Just for clarification: are we writing a test protocol for each PR now?

What do you mean by linking the implementation? 
Do you want a link to the folder where the feature is implemented?